docs/summarisinglogger.txt

```diff
@@ -84,16 +84,26 @@ The logging on script exit is done using python's :mod:`atexit`
 module. We can see that the handler we created above is now
 registered with this module by having a poke at its internals:
 
+>>> from mailinglogger.compat import PY2
 >>> import atexit
->>> print atexit._exithandlers
-[(<bound method SummarisingLogger.close of <...>, (), {})]
+>>> if PY2:
+...     len(atexit._exithandlers)
+... else:
+...     atexit._ncallbacks()
+1
```

Comment:
Well, okay, I see where you're going with this but the docs are there to show that a SummarisingLogger.close has been added as an exit handler, can we get that back?